### ISSUE TITLE
mouse moving to other applications with two monitors setup

### ISSUE THREAD

**Issue by foodornt** [WootingKb/wooting-double-movement#23]
When I turn on the double movement app and try to move the mouse in the game, and after playing for a while, the mouse will move out of the game screen and go to the other screen

**Comment by DarrenVs** [comment]
This is an issue unrelated to the Double Movement app, but rather a problem with how Fortnite handles the UI when you have a Gamepad and Mouse input at the same time.

The Double Movement app emulates a gamepad Joystick to create their specific running directions from digital keys.
When you open a menu interface like the Scoreboard or Inventory, your gamepad-movement and mouse inputs will fight on the UI whether to show Gamepad buttons or keyboard and mouse buttons.

The setting 'Lock Input Method as Mouse' is supposed to prevent this, but is likely not working as intended.
![image](https://user-images.githubusercontent.com/2780639/122673421-a6fe0480-d1d0-11eb-88aa-c66ab5a27103.png)

If you close your inventory, scoreboard, or other menu while the UI is in 'Gamepad Mode', your mouse will no longer get locked to the game window as a side effect to this problem.
## The only way to avoid this is by practicing some workarounds such as:
### Don't walk with a menu open to avoid any UI bugs
> You can still use the auto-run keybind to continue moving with the Inventory open
### Play with a single monitor enabled and fullscreen so the cursor can't leave the game window
> Shortcut: `Win` + `Shift` + `P` -> `PC screen only`

---
Bottom line;
Wooting can't fix this, make a Fortnite bug report [here](https://www.epicgames.com/help/en-US/fortnite-c75/technical-support-c118/how-do-i-submit-a-bug-report-in-fortnite-a3334) to gather some attention to this problem for the Fortnite developers.

**Comment by DarrenVs** [comment]
Alternative solution would be constraining the cursor to a specific window or monitor with other third party applications

**Comment by NateBoii98TV** [comment]
Very Often while I'm playing Fortnite (randomly in fights when editing, flick shot, etc. not just with map/inventory, etc up) go to my 2nd monitor even when I have no programs or windows open on my 2nd monitor. I've tried to emulate it and can't. I have my windows key cap off my keyboard, no other tabs open to alt/tab to. I don't understand. This software has caused many issues since I started using it. Push to talk on Fortnite doesn't work since, it doesn't recognize I'm holding my P2T bind. Often have turbo building issue, editing issues - most common scroll wheel reset, since the first day I used Wooting DM when scrolling to reset a build half the time it starts to edit/reset it will get stuck in edit mode before the reset is finale and/or it cancels out before it fully starts the edit/reset. It acts as if its being canceled out by another input. Same issue with turbo building, just much more rare.
